Sources/Valet/Valet.swift

```diff
@@ -285,6 +285,40 @@ public final class Valet: NSObject {
         }
     }
 
+    /// Convenience setter for `Codable` objects. Stores them using `JSONEncoder` and the `setObject(:forKey:)` method.
+    ///
+    /// - Parameters:
+    ///   - object: A `Codable` object to be inserted into the keychain.
+    ///   - key: A key that can be used to retrieve the `object` from the keychain.
+    ///   - jsonEncoder: The `JSONEncoder` to use for encoding the type to store. Defaults to the default initializer.
```

Comment:
Once we allow a generic encoder to be passed in, we'll need to change this comment and parameter name.